requirements/sample/umati/v3/metadata/example_publisher_1/uns/machines/FullMachineTool/2_Identification/5_SoftwareIdentification/OS/payload.json

```diff
@@ -0,0 +1,48 @@
+{
+    "Name": "nsu=http://example.com/FullMachineTool/;i=66382",
+    "Namespaces": [
```

Comment:
The Namespaces is used only in the this Message / the Data related to this MetaData 
Maybe we should request that all messages have the same namespace?